diff --git a/server/thread.c b/server/thread.c
index 7f0c62e..55960fc 100644
--- a/server/thread.c
+++ b/server/thread.c
@@ -670,10 +670,7 @@ static void set_thread_info( struct thread *thread,
         if ((req->priority >= min && req->priority <= max) ||
             req->priority == THREAD_PRIORITY_IDLE ||
             req->priority == THREAD_PRIORITY_TIME_CRITICAL)
-        {
             thread->priority = req->priority;
-            set_scheduler_priority( thread );
-        }
         else
             set_error( STATUS_INVALID_PARAMETER );
     }
diff --git a/server/thread.c b/server/thread.c
index 55960fc..0b9e703 100644
--- a/server/thread.c
+++ b/server/thread.c
@@ -48,10 +48,10 @@
 #include "process.h"
 #include "thread.h"
 #include "request.h"
-#include "user.h"
-#include "security.h"
 #include "esync.h"
 #include "fsync.h"
+#include "user.h"
+#include "security.h"
 
 
 /* thread queues */
